cluster/distvalidator.go

```diff
@@ -22,8 +22,13 @@ type DistValidator struct {
 	// PubKey is the root distributed public key.
 	PubKey string `json:"distributed_public_key"`
 
-	// Verifiers are the public shares.
-	Verifiers [][]byte `json:"threshold_verifiers"`
+	// PublicShares are the public keys of each node's secret key share.
+	// It can be used to verify a partial signature created by any node in the cluster.
+	PublicShares [][]byte `json:"public_shares,omitempty"`
+
+	// Verifiers are the threshold verifier commitments.
+	// Deprecated: Use PublicShares.
+	Verifiers [][]byte `json:"threshold_verifiers,omitempty"`
```

Comment:
discussion: why are we keeping this around then?

Reply:
still used, will refactor in another PR